{"title":"About InterMine"}

<p class="im-post-intro">
    InterMine is an open source data warehouse system for the integration and analysis of complex biological data,
    developed by the Micklem Lab at the University of Cambridge. InterMine has been used for developing data warehousing
    solutions for a number of projects, including for storage and analysis of modENCODE data, and as a data mining
    platform for a number of major model organism databases as part of the InterMOD project.

    InterMine has been principally funded by the Wellcome Trust [067205], [082598], [090297], [099133], [208381] with
    additional funding from the BBSRC [BB/L027151/1], [BB/P024335/1], Innovate UK [KTP011266] as well as support from
    the National Human Genome Research Institute.
</p>

<div class="im-post-section">
    <div class="im-post-section-para">
        The publicly available InterMine instances include:

        <ul class="im-post-ul">
            <li>
                {{<external-link href="https://www.humanmine.org" title="HumanMine">}} an integrated data warehouse of
                Homo sapiens genomic data
            </li>
            <li>
                {{<external-link href="https://www.flymine.org" title="FlyMine">}} a data warehouse of integrated
                fruit fly genetic, genomic and proteomic data
            </li>
            <li>
                {{<external-link href="http://yeastmine.yeastgenome.org" title="YeastMine">}} an integrated data
                warehouse of yeast genomic data, developed by SGD
            </li>
            <li>
                {{<external-link href="http://ratmine.mcw.edu/ratmine" title="RatMine">}} an integrated data warehouse
                of rat genomic data, developed by RGD
            </li>
            <li>
                {{<external-link href="http://www.mousemine.org" title="MouseMine">}} an integrated data warehouse of
                mouse genomic data, developed by MGI
            </li>
            <li>
                {{<external-link href="https://targetmine.mizuguchilab.org/targetmine" title="TargetMine">}} a data
                warehouse for candidate gene prioritisation and
            </li>
            <li>
                {{<external-link href="http://www.Zebrafishmine.org" title="Zebrafishmine">}} an integrated data
                warehouse of zebrafish genomic data, developed by
            </li>
            <li>
                {{<external-link href="https://bar.utoronto.ca/thalemine" title="Thalemine">}} a data warehouse for
                Arabidopsis thaliana Col-0 for the ARAPORT
            </li>
            <li>
                {{<external-link href="https://phytozome.jgi.doe.gov/phytomine" title="PhytoMine">}} an integrated
                data warehouse of over 50 plant genomes from
            </li>
        </ul>
        See the {{<external-link href="http://registry.intermine.org" title="InterMine registry">}} for the full list of
        InterMine instances.
    </div>
</div>

<div class="im-post-section">
    <div class="im-post-section-stack">
        <h2 class="im-post-section-heading">
            Legal
        </h2>

        <div class="im-post-section-para">
            All InterMine code is freely available under the open source
            {{<external-link href="http://www.gnu.org/licenses/lgpl.html" title="LGPL">}} license.
        </div>
    </div>
</div>